core/test/scenarios/__snapshots__/api-test-pptr.js.snap

```diff
@@ -258,6 +258,7 @@ Array [
   "largest-contentful-paint",
   "largest-contentful-paint-element",
   "layout-shift-elements",
+  "layout-shifts",
```

Comment:
Why is this just appearing now? Wouldn't this have been done in the previous PR?

Reply:
So looks like API test is failing on main for me locally. Seems like something weird is going on in CI so this PR LGTM